apps/src/userHeaderEventLogger/userHeaderEventLogger.js

```diff
@@ -1,16 +1,28 @@
 import $ from 'jquery';
 
-import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import {EVENTS, PLATFORMS} from '@cdo/apps/lib/util/AnalyticsConstants';
 import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import getScriptData from '@cdo/apps/util/getScriptData';
 
 $(document).ready(function () {
   const headerCreateMenu = document.getElementById('header_create_menu');
+  const pageUrl = document.querySelector('meta[property="og:url"]')?.content;
+
+  if (getScriptData('isSignedOut')) {
+    analyticsReporter.sendEvent(
+      EVENTS.SIGNED_OUT_USER_SEES_HEADER,
```

Comment:
<img width="350" alt="Screenshot 2024-06-07 at 10 23 22 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/31727474-d539-499b-9a6f-7030481ec013">

Reply:
Do we need to use the metadata to get the url? I think if you use `window.location.href` you can get the url regardless of pegasus vs. dashboard

Reply:
Ah that worked! Thanks for the tip 😄 

<img width="375" alt="Screenshot 2024-07-09 at 2 34 25 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/79509655-e92f-434a-b712-6ffc25f59bca">